src/repr/src/timestamp.rs

```diff
@@ -182,6 +198,54 @@ impl Timestamp {
     pub fn step_back(&self) -> Option<Self> {
         self.checked_sub(1)
     }
+
+    /// Rounds up the timestamp to the time of the next refresh, according to the given periodic
+    /// refresh schedule.
+    ///
+    /// Returns None if an overflow happens.
+    pub fn round_up(
+        &self,
+        RefreshEvery {
+            interval,
+            aligned_to,
+        }: &RefreshEvery,
+    ) -> Option<Self> {
+        // Planning ensured that
+        // - interval.months == 0, so we don't need to deal with months being of variable size.
+        // - The interval can be max 27 days, so the cast to u64 won't overflow.
+        // - The interval is positive, so the cast to u64 won't underflow.
```

Comment:
Instead of relying on specific planning behavior here, could you just document these cases in the `# Panics` section of this method and then let the planner worry about this methods requirements?

Reply:
Another requirement is that the interval (in milliseconds) must not be 0, because otherwise we'll get a division by zero panic.

Reply:
Done